docs/data-collection/tear-down.md

```diff
@@ -32,30 +33,21 @@
 - [ ] Give the participant the corresponding compensation for the participation and transportation.
 - [ ] Ask the participant to sign the receipt of the amount of the financial compensation.
 
-
 ## Clearing up the Scanning Room
 
 ### ET arm, ET cables, infrared mirror and stimuli screen
 
 - [ ] Unplug the two cables (signal and power) connected to the ET arm.
-    Put their extremities aside far from the scanner.
-- [ ] Agree with another experimenter, who will exit the room, sit on the other side of the access tubes in the Control Room, and pull cables (fiber and power of the ET) and tubes (RB and GA) out of the room.
-    At the same time, you carefully feed them so they do not suffer abrasion from touching the edges of the tube.
-
-    !!! warning "Extracting the cables and tubes requires two experimenters"
-
-- [ ] The experimenter currently outside in the Control Room will re-enter the Scanner Room with the plastic bag of the mirror and a fresh pair of gloves and will leave them prepared on the bed.
+- [ ] Roll the cables and put them in the cupboard inside the Scanning room.
 - [ ] Remove the mirror frame from its rails mounted on the head coil and lay it on the bed.
-- [ ] Put the gloves on and detach the infrared mirror from the standard mirror.
+- [ ] Put the gloves on and cover the infrared mirror for storage.
 
     !!! danger "The infrared mirror MUST be manipulated with clean gloves at all times."
 
-- [ ] Immediately insert the infrared mirror back into its plastic pouch.
-- [ ] Take the pouch containing the infrared mirror outside to the Control Room and place it back in its designated box, with extreme care.
-- [ ] Clean the standard mirror removing all residues of glue from the scotch tape.
-- [ ] Re-attach the mirror to the coil's frame.
+- [ ] Re-attach the standard mirror to the coil's frame.
```

Comment:
Usually, there is no standard mirror on the coil, so we can probably remove this step.

Reply:
```suggestion
```